deepspeed/runtime/engine.py

```diff
@@ -361,8 +361,10 @@ def __init__(self,
         self.flatten = _flatten_dense_tensors
         self.unflatten = _unflatten_dense_tensors
 
-        if self._config.compile_config.enabled:
-            self._set_client_model(CompiledModuleWrapper(self.module, self._config.compile_config))
+        self._is_compiled = False
+        self._compiler_backend = None
```

Comment:
```suggestion
        self._compiler_backend = get_backend_fn(self._config.compile_config.backend)
```

Reply:
Thank you, I also simplified setting _compiler_backend in forward